index.html

```diff
@@ -817,7 +819,7 @@ <h1 class="heading-bg">
         <section class="slide">
             <h2>Class exercise</h2>
             <ol>
-                <li>Add the box model fix to our CSS file. <a href="index.html#slide37">link</a></li>
+                <li>Add the box model fix to our CSS file. <a href="index.html#slide36">link</a></li>
```

Comment:
Previously linked to the funny gif slide. I think it was supposed to link to the code.